README.adoc

```diff
@@ -2,46 +2,68 @@
 
 This is the OpenShift Source to Image (S2I) image for OpenJDK.
 
+## How to get it
+
+### From https://access.redhat.com/containers/[Red Hat Container Catalog]
+
+This is the subscription supported product version.  It's but free and a subscription is not required to get the image.
```

Comment:
I'm not sure what the "legal" or support story is for accessing the product images w/o subscription. I think the lack of authentication on that registry is more of a technical-limitation/legacy issue than a policy one (not sure though, although the newer registry registry.redhat.io does require authentication).  So I'd prefer to drop the sentence " It’s but free and a subscription is not required to get the image."

Reply:
done